app/javascript/gobierto_data/lib/factories/data.js

```diff
@@ -2,7 +2,7 @@ import { baseUrl } from "../commons.js";
 import { getToken } from "../helpers";
 import axios from "axios";
 
-const endPoint = `${baseUrl}/data`;
+const endPoint = `${baseUrl}/data.csv`;
```

Comment:
If the endpoint returns a CSV, shouldn't the `"Content-type": "application/json"` (line 8) set as well accordingly? I ask, not sure. cc/ @ferblape @entantoencuanto

Reply:
The content-type returned for csv is `text/plain`. I seem to remember that we decided to use `text/plain` to avoid default browser behaviour of downloading a file instead of showing the content, but if required we can change the content type to `text/csv`